snappy_pipeline/workflows/somatic_variant_calling/model.py

```diff
@@ -248,3 +256,5 @@ class SomaticVariantCalling(SnappyStepModel, validators.ToolsMixin):
 
     varscan_joint: VarscanJoint | None = None
     """Configuration for VarscanJoint"""
+    tumor_normal_mode: str = TumorNormalMode.AUTOMATIC
```

Comment:
I think that this must be removed. You don't want two different definitions of the tumor/normal mode parameter